lib/PEX.ts

```diff
@@ -50,7 +50,7 @@ export class PEX {
    */
   public evaluatePresentation(
     presentationDefinition: IPresentationDefinition,
-    presentation: IPresentation | JwtDecodedVerifiablePresentation | string,
+    presentation: OriginalVerifiablePresentation,
```

Comment:
I am seeing these popup now:"

Argument of type 'IPresentation' is not assignable to parameter of type 'OriginalVerifiablePresentation'.

It probably has to todo with how we handle the uniform Presentations. So guess the 'IPresentation | JwtDecodedVerifiablePresentation | string`, needs to stay present, with string replaced by CompactJWT